sdk/tests/conformance2/renderbuffers/00_test_list.txt

```diff
@@ -1,3 +1,4 @@
+--min-version 2.0.1 framebuffer-completeness.html
```

Comment:
We can add framebuffer completeness related tests in this file and use different function names like runBaseLevelTest to differentiate them.  Does this make sense? If not, I can change the file name.